content/articles/ordering-a-wildcard-certificate.markdown

```diff
@@ -0,0 +1,47 @@
+---
+title: Ordering a Wildcard SSL Certificate
+excerpt: How to order a wildcard SSL certificate with DNSimple.
+categories:
+- SSL Certificates
+---
+
+# Ordering a Standard SSL Certificate
+
+### Table of Contents {#toc}
+
+* TOC
+{:toc}
+
+---
+
+Wildcard certificates allow you to secure all subdomains of a domain with a single certificate. Wildcard certificates can make certificate management easier in some cases. We still recommend non-wildcard certificates for most use cases.
+
+Wildcard certificates are no different to standard certificates, they only contain an `*` as the subdomain, so please make sure to check out the article about [Ordering a Standard Certificate](/articles/ordering-standard-certificate/) too.
```

Comment:
This sentence is potentially incorrect. It uses "standard certificate" as an alternative to "wildcard certificate". However, we've been using the term standard to refer to Comodo certificates.
See https://support.dnsimple.com/articles/standard-vs-letsencrypt/

What you probably wanted to use is single-name vs wildcard.

In fact, the counterpart of https://support.dnsimple.com/articles/ordering-standard-certificate/ is https://support.dnsimple.com/articles/ordering-lets-encrypt-certificate/